styleguide.config.js

```diff
@@ -92,21 +92,13 @@ module.exports = async () => {
 				name: 'Versions',
 				sections: [
 					{
-						name: 'next v9.x (Nextcloud 30+ on Vue 3)',
+						name: 'v9 (Nextcloud 30+ on Vue 3)',
```

Comment:
incomplete backport, on main its

```suggestion
						name: 'v9 (Nextcloud 31+ on Vue 3)',
```